exporter/azure_monitor/azure_monitor.go

```diff
@@ -3,60 +3,85 @@ package azure_monitor
 // This includes examples on how to create azure exporters to send spans.
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"log"
 
-	"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/exporter/azure_monitor/utils"
+	//"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/trace"
 )
 
-type AzureTraceExporter struct {
-	InstrumentationKey string
-	Options            common.Options
+// Exporter is an implementation of trace.Exporter that uploads spans to Azure Monitor.
+type Exporter struct {
+	ServiceName         string
+	InstrumentationKey  string
+	Context             context.Context
+	EndPoint            string
+	TimeOut             int
 }
 
-/*	Creates an Azure Trace Exporter.
-	@param options holds specific attributes for the new exporter
-	@return The exporter created and error if there is any
-*/
-func NewAzureTraceExporter(IKey string) (*AzureTraceExporter, error) {
-	if IKey == "" {
+// Options are the options to be used when initializing an Azure Monitor exporter.
+type Options struct {
+	ServiceName         string
+	InstrumentationKey  string	// required by user
+	Context             context.Context
+	EndPoint            string
+	TimeOut             int
+}
+
+// NewExporter returns an exporter that exports traces to Azure Monitor.
+func NewExporter(o Options) (*Exporter, error) {
+	if o.InstrumentationKey == "" {
 		return nil, errors.New("missing Instrumentation Key for Azure Exporter")
 	}
-	currentOptions := common.Options {
-		InstrumentationKey: IKey,
-		EndPoint:           "https://dc.services.visualstudio.com/v2/track",
-		TimeOut: 			10.0,
+	if o.EndPoint == "" {
+		o.EndPoint = "https://dc.services.visualstudio.com/v2/track"
 	}
-	exporter := &AzureTraceExporter {
-		InstrumentationKey: currentOptions.InstrumentationKey,
-		Options:            currentOptions,
+	if o.TimeOut == 0 {
+		o.TimeOut = 10.0
 	}
-	
-	return exporter, nil
+	e := &Exporter {
+        ServiceName:         o.ServiceName,
+        InstrumentationKey:  o.InstrumentationKey,
+        Context:             o.Context,

```

Comment:
What's the purpose of `Context` here?

Reply:
`Context` has no purpose here so I will remove it. I added it when I was following an example, but I never used it.